crates/subspace-node/src/chain_spec.rs

```diff
@@ -379,6 +385,14 @@ fn subspace_genesis_config(
         confirmation_depth_k,
     } = genesis_params;
 
+    let raw_domain_genesis_config = {
+        let mut domain_genesis_config = evm_chain_spec::get_testnet_genesis_by_spec_id(spec_id);
+        // Clear the WASM code of the genesis config since it is duplicated with `GenesisDomain::code`
+        domain_genesis_config.system = Default::default();
```

Comment:
This matches the comment precisely as system can possibly have other fields in the future.

```suggestion
        domain_genesis_config.system.code = Default::default();
```